### PR TITLE
[FIX] delivery: always assign the value

### PR DESCRIPTION
**Pull request by nim-odoo**
In order to prevent `CacheMiss` errors.

Closes #45242

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by nim-odoo** [comment]
robodoo r+

File: addons/delivery/models/sale_order.py [hunk 1]
```diff
@@ -143,7 +143,7 @@ class SaleOrderLine(models.Model):
     def _compute_product_qty(self):
         for line in self:
             if not line.product_id or not line.product_uom or not line.product_uom_qty:
-                continue
+                line.product_qty = 0.0
             line.product_qty = line.product_uom._compute_quantity(line.product_uom_qty, line.product_id.uom_id)
```
Comment (MiquelRForgeFlow) [inline addons/delivery/models/sale_order.py]:
```suggestion
                line.product_qty = 0.0
                continue
            line.product_qty = line.product_uom._compute_quantity(line.product_uom_qty, line.product_id.uom_id)
```
🤔

Comment (nim-odoo) [reply addons/delivery/models/sale_order.py]: indeed, thanks

**Comment by nim-odoo** [comment]
robodoo r+

**Pull request closed without merging**